nats/aio/client.py

```diff
@@ -230,7 +235,7 @@ def __init__(self) -> None:
         self._client_id: Optional[int] = None
         self._sid: int = 0
         self._subs: Dict[int, Subscription] = {}
-        self._status: int = Client.DISCONNECTED
```

Comment:
I would rather keep this to be backwards compatible just in case, I like the change into an enum but `Client.DISCONNECTED` etc... should still work